os/compression/compress_read.c

```diff
@@ -39,8 +39,24 @@
  * Private Declarations
  ****************************************************************************/
 
-struct s_header compression_header;
-struct s_buffer buffers;
+static struct s_header compression_header;
+
+/* Number of blocks for caching */
+static unsigned int number_blocks_caching;
+
+/* Block caches to be used for holding uncompressed blocks */
+static block_cache_t *blockcache;
+
+/* Pointers for maintaing doubly linked list of blockcaches */
```

Comment:
please change the maintaing -> maintaining

Reply:
done